src/main/io/vtx_rtc6705.c

```diff
@@ -232,6 +232,16 @@ static uint8_t vtxRTC6705GetPowerLevels(const vtxDevice_t *vtxDevice, uint16_t *
     return 0;
 }
 
+static void vtxRTC6705SerializeDeviceStatus(const vtxDevice_t *vtxDevice, sbuf_t *dst)
+{
+    const vtxDevType_e vtxType = vtxCommonGetDeviceType(vtxDevice);
+    const bool deviceReady = vtxCommonDeviceIsReady(vtxDevice);
+    sbufWriteU8(dst, 2);
```

Comment:
This will become self-explanatory (see also the implementation in all other protocols) if you go something like:

```
#define VTX_DEVICE_STATUS_SIZE 2

static void vtxRTC6705SerializeDeviceStatus(const vtxDevice_t *vtxDevice, sbuf_t *dst)
￼{
￼    const vtxDevType_e vtxType = vtxCommonGetDeviceType(vtxDevice);
￼    const bool deviceReady = vtxCommonDeviceIsReady(vtxDevice);
￼    sbufWriteU8(dst, VTX_DEVICE_STATUS_SIZE);
[...]
```

Reply:
@mikeller, I was about to replace message sizes with #defines. But then figured it's not going to work with SmartAudio, because it needs to be calculated in runtime:
`const uint8_t vtxDeviceStatusSize = 12 + powerLevelCount * 4;`
so I made `const` also for other protocols for consistency.
But let me know if you think its better to use #define for Tramp and rtc6705.